server/@types/viewModels/index.d.ts

```diff
@@ -71,11 +71,11 @@ declare module 'viewModels' {
     createdBy: string
     createdByDisplayName: string
     createdAt: Date
-    createdAtPrisonName?: string
```

Comment:
These fields always have a value (in the view model object) so we can remove the optionality which makes other code easier 👍